### DISCUSSION TITLE
Yolov5 on Nvidia Jetson TX2

### DISCUSSION THREAD

**Discussion by x-Surreal** [ultralytics/yolov5#3562] [General]
**How do I resolve this error?**

Traceback (most recent call last):
  File "detect.py", line 185, in <module>
    detect(opt=opt)
  File "/home/ktm/.local/lib/python3.7/site-packages/torch/autograd/grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "detect.py", line 54, in detect
    dataset = LoadImages(source, img_size=imgsz, stride=stride)
  File "/home/ktm/yolov5/utils/datasets.py", line 138, in _init_
    raise Exception(f'ERROR: {p} does not exist')
Exception: ERROR: /home/ktm/yolov5/../zidane.jpg does not exist

**Comment by x-Surreal** [comment]
**ran this piece of code** 
python3 detect.py --source ../zidane.jpg --weights yolov5s.pt --conf 0.25

**Reply by glenn-jocher** [reply]
use a --source that exists

**Reply by x-Surreal** [reply]
Getting an Assertion error for the same

**Comment by pravastacaraka** [comment]
@x-Surreal The `--source` argument you provided did not find the file. Try running this command `$ ls ../zidane.jpg` from the directory where you ran the python command above. See the results, is there a file called zidane.jpg?
